cassandra/tablets.py

```diff
@@ -0,0 +1,46 @@
+# Experimental, this interface and use may change
+import bisect
+from threading import Lock
+
+
+class Tablet(object):
+    """
+    Represents a single ScyllaDB tablet.
+    """
+    table_id = None
+    last_token = 0
+    tablet_count = 0
+    new_replicas = None
+    replicas = None
+    stage = ""
+
+    def __init__(self, table_id = None, last_token = 0, tablet_count = 0, new_replicas = None, replicas = None, stage = ""):
+        self.table_id = table_id
+        self.tablet_count = tablet_count
+        self.last_token = last_token
+        self.new_replicas = new_replicas
+        self.replicas = replicas
+        self.stage = stage
+
+    def __lt__(self, other):
+        return (self.last_token < other)
```

Comment:
I removed this operator and pass only last_tokens to `bisect_left`

Reply:
I'm sorry, I thought that `key` argument to `bisect_left` was already present in Python 3.6, but apparently it is not, my bad.

The current code will materialize whole list of `last_tokens` on each call to `get_tablet_for_key`, so it's not rhe right solution performance-wise.
I don't want to go back to `__lt__` for `Tablet < int`, it feels weird, I think better solution for now is to copy-paste `bisect_left` solution from Python 3.11: https://github.com/python/cpython/blob/fb08b7905eaf67902a89eeec0f8c1aa5ee0c7ea5/Lib/bisect.py#L68-L99 (with a comment explaining where it came from and why it is here) into `tablets.py` and call it. When we drop support for Python < 3.10, it will be just a matter of removing this impl and adjusting the call.